index.html

```diff
@@ -1,676 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why are we deleting basically everything here?

Reply:
This was the index.html file for the html based app. See public folder for index.html for react app.